chia/data_layer/s3_plugin_service.py

```diff
@@ -122,6 +125,20 @@ async def upload(self, request: web.Request) -> web.Response:
             return web.json_response({"uploaded": False})
         return web.json_response({"uploaded": True})
 
+    async def plugin_id(self, request: web.Request) -> web.Response:
+        return web.json_response(
+            {
+                "id": plugin_id,
```

Comment:
?

```suggestion
                "plugin_id": plugin_id,
```

Reply:
Hmm this looks wrong, I changed this to /`plugin_info`